src/LoadGoods.ts

```diff
@@ -0,0 +1,8 @@
+import { Good } from './types';
```

Comment:
it is not a Component, not a Class or Constructor, the file should be named from lowerCase letter